wildstore-cli/src/test/java/edu/sjsu/wildstore/CliTest.java

```diff
@@ -238,10 +235,16 @@ void testShare() throws IOException {
         result = clirun(cmd, "share", testDataPath.toString(), testDataPath2.toString(), "--metaURL", metaURL, "--token", userTokenFile.toString(), "--email", "user@share");
         Assertions.assertEquals(0, result.exitCode);
         Assertions.assertTrue(result.out.contains("?filename=test-data.txt") && result.out.contains("?filename=test-data-2.txt") && !result.err.contains("Missing Files"));
+        Query query = new Query(Criteria.where("fileDigest").is(testDigest));
+        LocalDateTime oldTime = Objects.requireNonNull(springCtx.getBean(MongoTemplate.class)
+                                                               .findOne(query, ShareLink.class, "share-links")).expiry;
 
-        result = clirun(cmd, "share", testDataPath.toString(), "/testfile", "--metaURL", metaURL, "--token", userTokenFile.toString(), "--email", "user@share");
+        result = clirun(cmd, "share", testDataPath.toString(), "/testfile", "--metaURL", metaURL, "--token", userTokenFile.toString(), "--email", "user@share", "--validFor", "year");
         Assertions.assertEquals(0, result.exitCode);
         Assertions.assertTrue(result.out.contains("?filename=test-data.txt") && result.err.contains("/testfile"));
+        LocalDateTime newTime = Objects.requireNonNull(springCtx.getBean(MongoTemplate.class)
+                                                               .findOne(query, ShareLink.class, "share-links")).expiry;
+        Assertions.assertTrue(newTime.isAfter(oldTime));
```

Comment:
can you add a test where you share testDataPath with --validFor day and make sure it stays a year?